src/System.Data.SqlClient/tests/Tools/TDS/TDS.EndPoint/SSPI/SecurityWrapper.cs

```diff
@@ -24,7 +24,7 @@ internal class SecurityWrapper
         /// <param name="pvGetKeyArgument">This parameter is not used and should be set to NULL</param>
         /// <param name="phCredential">A pointer to a CredHandle structure to receive the credential handle</param>
         /// <param name="ptsExpiry">A pointer to a TimeStamp structure that receives the time at which the returned credentials expire</param>
-        [DllImport(Interop.Libraries.SspiCli, CharSet = CharSet.Auto)]
+        [DllImport(Interop.Libraries.SspiCli, CharSet = CharSet.Auto, EntryPoint="AcquireCredentialsHandleW")]
```

Comment:
That is a good question, the only reason why I added the EntryPoint was because when ILCing this assembly, the entrypoint was not found so we couldn't PInvoke correctly, and it is now fixed with this. Do you suggest  I should reference the `AcquireCredentialsHandleA` function instead of the `AcquireCredentialsHandleW`? Same goes for the rest of your comments.